### PR TITLE
VB-8: Feature/db schema seeds

### PR DESCRIPTION
**Pull request by mishabenq**

### CODE REVIEW

File: packages/backend/prisma/seed/config/Categories.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export const Categories: string[] = [
+  "Ноутбуки та комп'ютери",
```
Comment (Vitalii4as) [inline packages/backend/prisma/seed/config/Categories.ts:2]: Use some constants and add categories to localization 
```
'LAPTOPS_AND_PC_CATEGORY_NAME',
```

In ua localization
```
LAPTOPS_AND_PC_CATEGORY_NAME: ''Ноутбуки та комп'ютери"
```

localization path packages/frontend/public/locales

File: packages/backend/prisma/seed/clearData.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { prismaClient } from './config/prismaClient';
+
+export const clearData = async () => {
+  try {
+    //clear
```
Comment (Vitalii4as) [inline packages/backend/prisma/seed/clearData.ts:5]: remove comment

File: packages/backend/prisma/seed/createData.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import { seedBids } from './controllers/bid';
+import { seedCategory } from './controllers/category';
+import { seedProducts } from './controllers/product';
+import { seedSocialMedia } from './controllers/socialMedia';
+import { seedUsers } from './controllers/user';
+import { seedUserSettings } from './controllers/userSettings';
+import { seedChatMember } from './controllers/chatMember';
+import { seedMessage } from './controllers/message';
+
+export const createData = async () => {
```
Comment (Vitalii4as) [inline packages/backend/prisma/seed/createData.ts:13]: generate data once and save to files

File: packages/backend/prisma/seed/createData.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  try {
+    await seedUsers(prismaClient);
+
+    //optimizing requests
```
Comment (Vitalii4as) [inline packages/backend/prisma/seed/createData.ts:17]: remove comment

**Review by Vitalii4as — DISMISSED** [review]

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**